examples/PSCoreApp/Profile.ps1

```diff
@@ -0,0 +1,20 @@
+# This script will be run on every COLD START of the Function App
+# You can define helper functions, run commands, or specify environment variables
+# NOTE: any variables defined that are not environment variables will get reset after the first execution
+
+# Example usecases for a Profile.ps1:
```

Comment:
I prefer to use `profile.ps1`, lower case.

Reply:
renamed